### ISSUE TITLE
Give project3 part 3 its own course?

### ISSUE THREAD

**Issue by junlong-gao** [talent-plan/tinykv#220]
I feel one simple section of the simple scheduler implementation cannot do justice to the marvelous topic of cluster monitoring, load balancing, and directory/discovery service as implemented in the current placement driver. I somehow hacked the proto files so that up to project 3b, the entire setup can be bootstrapped by a version of the official placement driver (no further testing is done except writing and reading a few keys using raw kv api).

Maybe it is worth considering having a separate course for the placement driver and use the official pd proto def here? Better yet people can start playing around tinykv after finishing project 3 and use an official pd implementation.

**Comment by Connor1996** [comment]
Yeah, It was planned to make the scheduler implementation as an individual repo just like tinykv. But due to limited resources and time, and considering the complexity of the whole project, it was made to be simple. 

Good idea for having a separate course. We may consider it in the future.
